src/www/interfaces_vlan_edit.php

```diff
@@ -200,6 +206,31 @@
                       </div>
                     </td>
                   </tr>
+<?
+$priorities = array(
+    1   => gettext('1 - Background'),
+    0   => gettext('0 - Best Effort (default)'),
```

Comment:
I agree but the browser does not care and if the user isn't explicitly selecting the priority he sets a lower priority. think about users who edit their old vlan id and hit save..